### DISCUSSION TITLE
Break line for Label in Forms

### DISCUSSION THREAD

**Discussion by kasgithub8** [Budibase/budibase#6245] [Help] [answered]
Hey,

is there any way to insert a break line for labels in forms?
Problem is if you have some words that need to be in the label, 
it doesn't look good because its just stretching every other form with it.

Thanks!

PS: \n & br don't work

**Comment by melohagan** [comment, marked as answer]
Hi @kasgithub8 

I definitely see what you mean. 
I've raised an issue here: https://github.com/Budibase/budibase/issues/6249

I was able to workaround this using an embed style: https://docs.budibase.com/docs/custom-css#global-styling

**CSS**
```
<style>
label {
min-width: 300px !important;
white-space: normal !important;
}
.spectrum-Textfield-input {
min-width: 300px !important;
}
</style>
```

![Screenshot 2022-06-09 at 12 07 26](https://user-images.githubusercontent.com/101575380/172833085-64ae4a5b-51e5-4919-b26d-993133f8532d.png)

It is a bit clunky though.

**Reply by kasgithub8** [reply]
Hi @melohagan, 

thanks for the quick answer and raising an issue. Where did u put the CSS? If I put it on my Options Picker, nothing changes.

Thanks!

**Reply by melohagan** [reply]
It goes inside the embed component: https://docs.budibase.com/docs/embed

**Reply by kasgithub8** [reply]
And you add the embed component to the Layout or?

**Reply by melohagan** [reply]
It can go anywhere, it applies the style to the entire page no matter where it is placed. I tend to put it at the top of my component tree to keep it out of the way.

**Reply by kasgithub8** [reply]
Damn, won't work for me.

**Reply by melohagan** [reply]
Did it not work? Or is it that the global styles wont be suitable for your use case?

**Reply by kasgithub8** [reply]
It just doesn't work for me. I tried the embed component everywhere with the CSS.

**Reply by melohagan** [reply]
And the embed text includes the style tags? If you're still having issues, feel free to send your app export to support@budibase.com and I can have a look.

**Reply by kasgithub8** [reply]
I see, I got it to work now. Now I noticed that your solution is not suitable for my use case. I think I really need the line break.
